docs/source/en/guides/collections.md

```diff
@@ -0,0 +1,187 @@
+   'author': 'TheBloke',
+   'item_id': 'TheBloke/TigerBot-70B-Chat-GPTQ',
+   'item_type': 'model',
+   'lastModified': '2023-09-19T12:55:21.000Z',
+   'position': 0,
+   'private': False,
+   'repoType': 'model'
+   (...)
+  }
+}
+```
+
+The [`Collection`] object returned by [`get_collection`] contains:
+- high-level metadata: `slug`, `owner`, `title`, `description`, etc.
+- a list of [`CollectionItem`] objects. Each item represents a model, a dataset, a Space or a paper.
+
+All items of a collection are guaranteed to have:
+- a unique `item_object_id`: this is the id of the collection item in the database
+- an `item_id`: this is the id on the Hub of the underlying item (model, dataset, Space, paper). It is not necessarily unique! only the `item_id`/`item_type` pair is unique.
+- an `item_type`: model, dataset, Space, paper.
+- the `position` of the item in the collection. Position can be updated to re-organize your collection (see [`update_collection_item`] below)
+
+A `note` can also be attached to the item. This is useful to add additional information about the item (e.g. a comment, a link to a blog post, etc.). If an item doesn't have a note, the attribute still exists with a `None` value.
+
+In addition to these base attributes, returned items can have additional attributes depending on their type: `author`, `private`, `lastModified`, `gated`, `title`, `likes`, `upvotes`, etc. None of these attributes are guaranteed to be returned.
+
+## Create a new collection
+
+Now that we know how to get a [`Collection`], let's create our own! Use [`create_collection`] with a title and optionally a description.
+
+```py
+>>> from huggingface_hub import create_collection
+
+>>> collection = create_collection(
+...     title="ICCV 2023",
+...     description="Portfolio of models, papers and demos I presented at ICCV 2023",
+... )
+```
+
+It will return a [`Collection`] object with the high-level metadata (title, description, owner, etc.) and an empty list of items. You will now be able to refer to this collection using it's `slug`.
+
+```py
+>>> collection.slug
+'iccv-2023-15e23b46cb98efca45'
+>>> collection.title
+"ICCV 2023"
+>>> collection.owner
+"username"
+```
+
+To create a collection on an organization page, pass `namespace="my-cool-org"` when creating the collection. Finally, you can also create private collections by passing `private=True`.
+
+## Manage items in a collection
+
+Now that we have a [`Collection`], we want to add items to it and organize them.
+
+### Add items
+
+Items have to be added one by one using [`add_collection_item`]. You only need to know the `collection_slug`, `item_id` and `item_type`. Optionally, you can also add a `note` to the item.
+
+```py
+>>> from huggingface_hub import create_collection, add_collection_item
+
+>>> collection = create_collection(title="OS Week Highlights - Sept 18 - 24", namespace="osanseviero")
+>>> collection.slug
+"osanseviero/os-week-highlights-sept-18-24-650bfed7f795a59f491afb80"
```

Comment:
nice find :D

Reply:
Always try to highlight some cool stuff in the docs examples :hugs:

Reply:
nice growth hack for Omar's profile:)